test/framework/config.go

```diff
@@ -150,6 +151,13 @@ func (c E2eConfig) AutoConfigure() error {
 		Config.CIDR = "fd00:fd00::/64"
 	}
 
+	if Config.KumaDeltaKDS {
+		Config.KumaCpConfig.Multizone.KubeZone1.Envs["KUMA_EXPERIMENTAL_KDS_DELTA_ENABLED"] = "true"
+		Config.KumaCpConfig.Multizone.KubeZone2.Envs["KUMA_EXPERIMENTAL_KDS_DELTA_ENABLED"] = "true"
+		Config.KumaCpConfig.Multizone.UniZone1.Envs["KUMA_EXPERIMENTAL_KDS_DELTA_ENABLED"] = "true"
+		Config.KumaCpConfig.Multizone.UniZone2.Envs["KUMA_EXPERIMENTAL_KDS_DELTA_ENABLED"] = "true"
+	}
+
```

Comment:
should we run a mixture of experimental and old KDS?

Reply:
It sounds more like a different use case. I would like to run just the new KDS to ensure that everything works correctly in the delta KDS.